OloEditor/src/EditorLayer.cpp

```
@@ -101,11 +101,14 @@ namespace OloEngine
 
         Application::Get().GetWindow().SetTitle("Test");
 
-        m_IconPlay = Texture2D::Create("Resources/Icons/PlayButton.png");
-        m_IconPause = Texture2D::Create("Resources/Icons/PauseButton.png");
-        m_IconSimulate = Texture2D::Create("Resources/Icons/SimulateButton.png");
-        m_IconStep = Texture2D::Create("Resources/Icons/StepButton.png");
-        m_IconStop = Texture2D::Create("Resources/Icons/StopButton.png");
+        // Toolbar icons are authored colour bitmaps — load as sRGB so the GPU
+        // linearises them on sample, matching how every other colour texture
```

Cppcheck (2.20.0)

[style] The function 'IsBuildInProgress' is never used.

(unusedFunction)